spec/game_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../lib/game.rb'
+require_relative '../lib/player.rb'
+
+describe Game do
+  let(:player_one) { Player.new('Hassan', 'X') }
+  let(:player_two) { Player.new('Jocyline', 'O') }
+  let(:game) { Game.new(player_one, player_two) }
+  describe '#finished?' do
+    it 'should return true when there is a winner or no available movies' do
```

Comment:
* [ ] Let us make this test cover the negative scenario.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
 ie ``` it 'should return false when there is a no winner but there are available moves ```
kindly rectify the spelling error as well it is misleading.